src/lib/message/L1Transaction.ts

```diff
@@ -98,6 +98,18 @@ export class L1TransactionReceipt implements TransactionReceipt {
     this.status = tx.status
   }
 
+  /**
+   * Check if is a classic transaction
+   * @param l2SignerOrProvider
+   */
+  public async isClassic<T extends SignerOrProvider>(
```

Comment:
We should call this function inside `getL1ToL2MessagesClassic` and `getL1ToL2Messages`